### ISSUE TITLE
mount module with the state=unmounted not removing entries from /etc/fstab

### ISSUE THREAD

**Issue by sivakom** [ansible/ansible-modules-core#2623]
**Issue Type:**
    Bug Report

**Ansible Version:**
Running ansible 1.9.2-bbg (detached HEAD 951fac4181). I did try it on the latest version but the issue still persists.

**Ansible Configuration:**
Haven't modified the `ansible.cfg` file or any configuration regarding ansible.

**Environment:**
Running it on Ubuntu 14.04.3 LTS.

**Summary:**
I'm trying to unmount a device in an ordinary linux machine with the ansible mount module. Let's assume that the following is an example of a device that I am trying to unmount (lsblk output): 
`sdc1   8:33   0   3.7T  0 part /tmp/sdc1`

When I run this appropriate ansible command:
`- mount: name=/tmp/sdc1 src=/dev/sdc1 fstype=xfs state=unmounted`

This does unmount the device but doesn't remove the entry from the `/etc/fstab` file. According to the docs, "if mounted or unmounted, the device will be actively mounted or unmounted as needed and _appropriately configured in fstab_", the fstab entry should be removed, but is clearly not being so. However, if I set the state to `absent`, the entry is indeed deleted but this necessitates using another task just to ensure this behavior.

**Steps To Reproduce:**
Firstly, there needs to be a linux device with a mountpoint which also has an associated entry in the `/etc/fstab` file. Then you can run the following task:
`- mount: name=<mountpoint> src=<device> fstype=<device's file system type> state=unmounted`

The device should then be unmounted afterwards, a simple run of `lsblk`should do. But if you look into the contents of `/etc/fstab`, the device's fstab entry should still be there.

**Expected Results:**
As stated before, when running the mount module with a `state=unmounted`, the device should be unmounted and have it's entry removed from the `/etc/fstab` file.

**Actual Results:**
Device is unmounted but the entry remains in `/etc/fstab`.

**Comment by jimi-c** [comment]
I believe this is a documentation issue, and not a bug, as `absent` is clearly the behavior you want.

**Comment by ansibot** [comment]
@grossws, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@grossws, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
This issue was migrated to https://github.com/ansible/ansible/issues/29982
